### DISCUSSION TITLE
Shared memory

### DISCUSSION THREAD

**Discussion by DincyFrancis** [eProsima/Fast-DDS#5130] [Q&A]
how fragment size selection happening for shared memory?

**Comment by JesusPoderoso** [comment]
Hi @DincyFrancis, thanks for using Fast DDS.
Please refer to the [Shared Memory section](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/shared_memory/shared_memory.html#) of the Fast DDS documentation, in particular to the [transport descriptor](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/shared_memory/shared_memory.html#sharedmemtransportdescriptor) `segment_size` configuration.
I am moving this discussion to the [Fast DDS](https://github.com/eProsima/Fast-DDS/discussions/categories/q-a) Q/A section.
